sdk/servicebus/service-bus/samples/browserSample.html

```diff
@@ -9,19 +9,19 @@
 <script src="../browser/service-bus.js"></script>
 
 <script>
-  // SBC holds reference to the ServiceBusClient global object exposed by the imported script
-  const SBC = Azure.Messaging.ServiceBus.ServiceBusClient;
+  // ServiceBusClient holds reference to the ServiceBusClient global object exposed by the imported script
+  const ServiceBusClient = Azure.Messaging.ServiceBus.ServiceBusClient;
 
   const connectionString = "";
   const queueName = "";
 
   async function main() {
     // Creates the instance of ServiceBusClient using input connection string
     // Refer to SDK documentation and other Javascript samples for more usage related information
-    const serviceBusClientObj = SBC.createFromConnectionString(connectionString);
+    const sbClient = ServiceBusClient.createFromConnectionString(connectionString);
 
     try {
-      const client = serviceBusClientObj.createQueueClient(queueName);
+      const client = sbClient.createQueueClient(queueName);
```

Comment:
Now that we have the word "client" in "sbClient", the other "client" gets confusing.
Can we replace `client` -> `queueClient` or `topicClient` or `subscriptionClient` as appropriate?

Reply:
These can be updated in stress and perf tests too where applicable

Reply:
Currently, we are concentrating on just the samples as we expect more traffic there due to the release of the GA version

Reply:
Test files are internal, so can be touched as and when we make the next of changes there